pkg/tsdb/influxdb/model_parser.go

```diff
@@ -40,7 +41,7 @@ func (qp *InfluxdbQueryParser) Parse(model *simplejson.Json, dsInfo *models.Data
 		return nil, err
 	}
 
-	parsedInterval, err := interval.GetIntervalFrom(dsInfo, model, time.Millisecond*1)
+	parsedInterval, err := GetIntervalFrom(dsInfo, model, time.Millisecond*1)
```

Comment:
Given @dsotirakis has merged his Elasticsearch changes this should use `tsdb.GetIntervalFrom` now was my impression?

Reply:
yes, I will use the tsdb one